src/main.js

```diff
@@ -65,4 +71,21 @@ function clickHomeButton() {
   document.querySelector('.random-cover-button').classList.remove('hidden');
   document.querySelector('.home-button').classList.add('hidden');
   document.querySelector('.form-view').classList.add('hidden');
+  displayCustomCover(currentCover);
+};
+
+function clickMakeMyBookButton() {
+  covers.push(coverInput.value);
+  titles.push(titleInput.value);
+  descriptors.push(descriptor1Input.value);
+  descriptors.push(descriptor2Input.value);
+  currentCover = new Cover(coverInput.value, titleInput.value, descriptor1Input.value, descriptor2Input.value);
```

Comment:
Lines 78 - 81 are doing one thing, and line 82 is doing another. Could you possibly separate some of this out into another function?